corona-warn-app/cwa-app-android
# Fix exposure logging progress circle inconsistencies. (EXPOSUREAPP-1835)

This PR makes the circle progress in line with the iOS one. Adding the x/14 days inside the circle and changing the progress color depending on the state of the exposure logging.

How to test:
- Go to exposure logging and look at the number of days inside the progress circle. It should look like x/14 days now.
- Try playing with the exposure logging states. Stop it, turn it back on again. Make sure that the circle is blue while logging is on and grey while it's off.
- I've removed two layers of includes in the fragment as they were only being used in this screen. Make sure the layout hasn't changed. Try disabling bluetooth and make sure the right cards appear on the screen.
